heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
```

Sorry, this one is definitely confusing and is apparently a bug in the CHEESEHub UI 😅 

To display this in the public catalog, we should set `display=stack`:
```suggestion
    "display": "stack",
```

This is supposed to be the default here, but the JSON appears to have the wrong value
I think that `standalone` is for some advanced functionality that is not used by CHEESEHub (optional dependencies)